### PR TITLE
Resolver award photos

### PR DESCRIPTION
**Pull request by deboer-tim**
Adds support for showing an award for multiple teams as a set of photos instead of a list.

### CODE REVIEW

File: PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+			if (numTeams > 25)
+				numColumns = 6;
+			else if (numTeams > 16)
+				numColumns = 5;
+			else if (numTeams > 9)
+				numColumns = 4;
+			else if (numTeams > 4)
+				numColumns = 3;
+			else if (numTeams > 1)
+				numColumns = 2;
```
Comment (nickygerritsen) [inline PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java:75]: How about
```suggestion
			numColumns = 1;
			if (numTeams > 1)
				numColumns = Math.ceil(Math.sqrt(numTeams) + 1);
```

File: PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+		numColumns = 1;
+		if (numTeams > 25)
+			numColumns = 6;
+		else if (numTeams > 16)
+			numColumns = 5;
+		else if (numTeams > 9)
+			numColumns = 4;
+		else if (numTeams > 4)
+			numColumns = 3;
+		else if (numTeams > 1)
```
Comment (nickygerritsen) [inline PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java:156]: Here as well

**Review by nickygerritsen — APPROVED** [review]

File: PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java (hunk 2 above)
Comment (deboer-tim) [reply PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java]: Fixed.

File: PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java (hunk 1 above)
Comment (deboer-tim) [reply PresContest/src/org/icpc/tools/presentation/contest/internal/presentations/resolver/TeamListPhotoPresentation.java]: Fixed.

**Pull request merged**